src/components/explorer/IndividualTracks.js

```diff
@@ -194,11 +194,11 @@ const IndividualTracks = ({individual}) => {
 
     return (
         <>
-        {Boolean(allFoundFiles.length) && (
+        {Boolean(allFoundFiles.length) ? (
```

Comment:
Just an innocent question about a part of the code that was already there: why is there a creation of a Boolean object here? Is that considered a best practise for legibility?

(As a personal preference here, I might have used `> 0` had I wanted the predicate to evaluate as a boolean)